kolibri/plugins/management/assets/src/views/user-page/user-create-modal.vue

```diff
@@ -119,61 +117,70 @@
       LEARNER: () => UserKinds.LEARNER,
       COACH: () => UserKinds.COACH,
       ADMIN: () => UserKinds.ADMIN,
-      statusMessage() {
-        if (this.errorMessage) {
-          return this.errorMessage;
-        } else if (this.confirmationMessage) {
-          return this.confirmationMessage;
+      usernameAlreadyExists() {
+        return this.users.find(user => user.username === this.username);
```

Comment:
I think it has to do with Keen - it doesn't seem to support it, but I'm not sure why.